src/app/services/data.service.ts

```diff
@@ -189,7 +362,7 @@ export class DataService {
     this.setStorage();
   }
 
-  public getCo2(): Co2 {
+  public getCo2PersonalGoal(): Co2 {
```

Comment:
is the set function renamed accordingly?

Reply:
doh...